docs/src/genericfunctions.md

```diff
@@ -0,0 +1,10 @@
+# Generic interface
+
+Any custom implementation with elements and parents should implement the
+following interface:
+
+```@docs
+parent(::Perm)
+elem_type(::SymmetricGroup)
+parent_type(::Perm)
```

Comment:
Shouldn't these signatures have their types unrestricted?
For `parent`, IIUC the problem you mention in the OP, I think the "docs/make.jl" file is already configured to tell `Documenter` to include only docstrings created in `AbstractAlgebra` via the `modules` keyword of the `makedocs` function (https://juliadocs.github.io/Documenter.jl/stable/man/guide/#Filtering-included-docstrings).

Reply:
Yes, I see that I forgot to push my last commit. Will push it when I get back to my main PC.

And I was not aware of that setting, thank you.